test/test_ops.py

```diff
@@ -160,6 +161,34 @@ def test_numpy_ref(self, device, dtype, op):
         finally:
             torch.set_default_dtype(cur_default)
 
+    # Tests that the cpu and gpu results are consistent
+    @onlyCUDA
+    @suppress_warnings
+    @ops(_ops_and_refs_with_no_numpy_ref, dtypes=OpDTypes.any_one)
+    def test_compare_cpu(self, device, dtype, op):
+
+        if dtype not in op.dtypes:
+            raise unittest.SkipTest("This test requires both cuda and cpu support")
+        if dtype not in op.dtypesIfCUDA:
+            raise unittest.SkipTest("This test requires both cuda and cpu support")
+
+        def to_cpu(arg):
+            if isinstance(arg, torch.dtype):
+                return arg
+            return arg.to(device='cpu')
```

Comment:
This is a very intriguing function. Are the arguments passed to this function just `dtype` or `Tensor`? In any case, a more natural way of writing this I think it'd be
```suggestion
        def to_cpu(arg):
            if isinstance(arg, Tensor):
                return arg.to(device="cpu")
            return arg
```

Reply:
updated